### ISSUE TITLE
Cannot select "@class" in projection of embedded property

### ISSUE THREAD

**Issue by creisle** [orientechnologies/orientdb#8926]
### Java Version: 
openjdk version "1.8.0_191"
OpenJDK Runtime Environment (build 1.8.0_191-b12)
OpenJDK 64-Bit Server VM (build 25.191-b12, mixed mode)
## Expected behavior  

Expected the `@class` attribute to be select-able like it is for non-embedded properties

## Actual behavior  
Always returns null

## Steps to reproduce  

```sql
create class Parent extends V
create class Child extends V
create property Parent.child EMBEDDED Child
create vertex parent set child = {"@class": "child", "name": "alice"}
```

Now when we select the record without explicitly accessing child.@class everything looks as expected

```sql
select * from parent
```

```json
{
    "result": [
        {
            "@type": "d",
            "@rid": "#15:0",
            "@version": 1,
            "@class": "Parent",
            "child": {
                "@type": "d",
                "@version": 0,
                "@class": "child",
                "name": "alice"
            }
        }
    ],
}
```

But when we try to filter on child.@class or just retrieve the value at the top level it is always null

```sql
select child.@class from parent
```

```json
{
    "result": [
        {
            "child.@class": null
        }
    ],
}
```

**Comment by luigidellaquila** [comment]
Hi @creisle 

I managed to reproduce the case, I'd say it'a s bug.
I'll fix it ASAP and keep you updated

Thanks

Luigi

**Comment by luigidellaquila** [comment]
Hi @creisle 

I just pushed a fix, it will be in the snapshot in a few minutes and will be released with v 3.0.22

Thanks

Luigi

**Comment by creisle** [comment]
Great! thanks for the quick fix :)
